kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSessionImpl.scala

```diff
@@ -113,6 +113,11 @@ class KyuubiBatchSessionImpl(
       ms.incCount(MetricRegistry.name(CONN_OPEN, user))
     }
 
+    checkSessionAccessPathURIs()
+
+    // we should call super.open before running batch job submission operation
```

Comment:
the comment explained what the code does, but not WHY should we do that, it's more important.

Reply:
updated